test/index.js

```diff
@@ -386,10 +386,9 @@ test('ChatList methods', t => {
 })
 
 test('key transfer methods', t => {
-  t.comment('test234')
-  dc.initiateKeyTransfer(setupCode => {
+  dc.initiateKeyTransfer((err, setupCode) => {
+    t.ok(err == null, 'err is not null')
```

Comment:
Better to do `t.same(err, null, 'err is not null')`

Reply:
The message as last parameter should say what the result should be, not what it shouldn't be.